apps/src/templates/census2017/CensusForm.jsx

```diff
@@ -99,15 +116,49 @@ class CensusForm extends Component {
     });
   }
 
-  toggle(option) {
-    const selected = this.state.selected.slice(0);
+  togglePledge() {
+    this.setState({
+      submission: {
+        ...this.state.submission,
+        acceptedPledge: !this.state.submission.acceptedPledge
+      }
+    });
+  }
+
+  toggleHowMuchCS(option) {
+    const selected = this.state.selectedHowMuchCS.slice(0);
     if (selected.includes(option)) {
+      const newSelected = _.without(selected, option);
       this.setState({
-        selected: _.without(selected, option)
+        selectedHowMuchCS: newSelected,
+        showFollowUp: this.checkShowFollowUp(newSelected)
       });
     } else {
+      const newSelected = selected.concat(option);
       this.setState({
-        selected: selected.concat(option)
+        selectedHowMuchCS: newSelected,
+        showFollowUp: this.checkShowFollowUp(newSelected)
+      });
+    }
+  }
+
+  checkShowFollowUp(selected) {
+    if (selected.includes("twenty_hr_some_b") || selected.includes("twenty_hr_all_b")) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
I think this can just be `selected.includes("twenty_hr_some_b") || selected.includes("twenty_hr_all_b")` ?